dsp/window/window_test.go

```diff
@@ -190,7 +218,7 @@ func TestGausWindows(t *testing.T) {
 				src[i] = 1
 			}
 
-			dst = NewValues(gaussian.Transform, len(src)).Transform(src)
+			dst = NewValues(trans.Transform, len(src)).Transform(src)
 			if !floats.EqualApprox(dst, test.want, tol) {
 				t.Errorf("unexpected result for lookup window function %q:\ngot:%#.6v\nwant:%#.6v", test.name, dst, test.want)
```

Comment:
This should have a verb for the window type. Given that, maybe the look up above should be `map[string]transformer` and the test struct's window type field should be `string`.